src/MembershipsSDKAdapter.js

```diff
@@ -37,6 +56,125 @@ export default class MembershipsSDKAdapter extends MembershipsAdapter {
     super(datasource);
 
     this.members$ = {}; // cache membership observables based on membership id
+    this.listenerCount = 0;
+  }
+
+  /**
+   * Tells the SDK to start listening to memberships events and tracks the amount of calls.
+   *
+   * Note: Since the SDK listens to ALL memberships events, this function only
+   * calls the SDK's `memberships.listen` function on the first membership to listen.
+   * Repeated calls to `memberships.listen` are not needed afterwards.
+   *
+   * @private
+   */
+  startListeningToMembershipsUpdates() {
```

Comment:
Do you think it would be a good idea to follow the MeetingsAdapter pattern of having a `connect` and `disconnect` method than then gets called with [`WebexSDKAdapter.connect()` as part of the global setup](https://github.com/webex/sdk-component-adapter/blob/master/src/WebexSDKAdapter.js#L26) instead of keeping a counter?

Reply:
Using the same approach as for the `MeetingsAdapter` the WebSocket will start emitting events right after we call `connect()` on the adapter but for example, we don't have any subscribers to the `getMembersFromDestination()`. I think this approach will just keep the WebSocket Network busy with events that maybe we don't want at that point.

While using a counter in `MembershipsAdapter` we will start listening to events only when someone subscribes to `getMembersFromDestination()`.